scripts/pkg.pr.new-custom-comment.mjs

```diff
@@ -0,0 +1,116 @@
+import LZString from 'lz-string';
+
+/**
+ * Used in `/.github/workflows/pkg.pr.new.yml`
+ */
+export async function postCustomComment({ github, context, output }) {
+	// eslint-disable-next-line no-console -- For debugging on github actions.
+	console.log('pkg-pr-new publish output:', JSON.stringify(output));
```

Comment:
@jeddy3 should we allow `debug` [here](https://github.com/stylelint/eslint-config-stylelint/blob/main/index.js#L37)?